src/torrijas.md

```diff
@@ -0,0 +1,42 @@
+# Torrijas (A spanish eastern treat)
+
+Traditional Spanish dessert, consumed especially during the catholic season called "Lent". As a curiosity, they used to be given as a gift to parturient women. You will have delicious sweet toasts.  There are many torrijas recipes, but this one is a keeper.
```

Comment:
```suggestion
# Torrijas (A spanish eastern treat)

![torrijas](pix/torijas.webp)

Traditional Spanish dessert, consumed especially during the catholic season called "Lent". As a curiosity, they used to be given as a gift to parturient women. You will have delicious sweet toasts.  There are many torrijas recipes, but this one is a keeper.
```

Image added in. You have to accept this though. I don't have the privileges to change your PRs for you.